components/nimbus/src/tests/test_behavior.rs

```diff
@@ -1278,8 +1281,13 @@ mod event_store_tests {
             1.0,
             store.query(event_id, Interval::Days, 56, 0, EventQueryType::LastSeen)?
         );
+
         assert_eq!(
-            0.0,
+            if now.weekday().number_from_monday() < then.weekday().number_from_monday() {
+                0.0
+            } else {
+                1.0
+            },
```

Comment:
This, on reflection, is definitely wrong, and will fail tomorrow.

Reply:
Ah dang